content/en/platform/corda/4.3/enterprise/node-operations-upgrade-cordapps.md

```diff
@@ -72,6 +72,13 @@ In order to perform the upgrade, follow the following steps:
     * By RPC using the `stateMachinesSnapshot` method and checking that there are no results
     * Via the shell by issuing the following command `run stateMachinesSnapshot`
 
+    {{< note >}}
+
+    The `stateMachinesSnapshot` method ties the flow ID with the `flowName` class. The result shows you all the flows running in your node.
+    You can use the `flow watch` method instead. It omits all the details returned by `stateMachinesSnapshot` that you don't need.
```

Comment:
Would it be better to say "You can use the `flow watch` method instead, which omits all the details returned by `stateMachineSnapshot` that you dont need"?

Reply:
Will send the note for non-technical review separately. Leaving as is for now.